packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -82,20 +100,66 @@ class Carousel extends Foundation<
      */
     public render(): React.ReactElement<HTMLDivElement> {
         return (
-            <div {...this.unhandledProps()} className={this.generateClassNames()}>
+            <div
+                {...this.unhandledProps()}
```

Comment:
I believe this should be below all of the declared props otherwise you risk overriding ref, onFocus etc.

Reply:
In terms of unhandled? Unhandled needs to be first so we ensure we don’t override those. If I pass this after the unhandled props will blow the ref and onFocus away.

Reply:
I had that backwards in terms of overriding then